tests/exploreServicesBreakDown.spec.ts

```diff
@@ -184,7 +184,7 @@ test.describe('explore services breakdown page', () => {
     await expect(table.getByRole('columnheader').nth(1)).toContainText('body');
 
     // Open the menu for "Line"
-    await page.getByRole('button', { name: 'Show menu' }).nth(1).click();
+    await page.getByLabel(/Show body|Line menu/).click();
```

Comment:
Not 100% sure I understand this change. We don't know at this point if the field name is going to be body or Line?

Reply:
Yeah, without dataplane flag the Line field name is body

Reply:
Right! Good call.